docs/content/management-bundles.md

```diff
@@ -252,6 +252,189 @@ and number keys are converted to strings. Also, YAML !!binary tags are not
 supported.
 {{< /info >}}
 
+### Bundle Container Format
```

Comment:
This section no longer fits in. For "Bundle Container Format", it made sense to have it follow "Bundle File Format", but what we really need is a separate OCI section at the bottom the bundle docs.

💭 Maybe it would make more sense to have an "OCI Registry" section next to the other "Implementations", like, following Nginx.